controllers/cluster_controller.go

```diff
@@ -178,6 +180,7 @@ func (r *ClusterReconciler) SetupWithManager(mgr ctrl.Manager, log logr.Logger)
 }
 
 // Reconcile reconciles a cluster object.
+// nolint:gocyclo //TODO: Reduce high cycomatic complexity.
```

Comment:
we might want a ticket for this one
or else we'll never do it

Reply:
Created the ticket [here](https://github.com/aws/eks-anywhere-internal/issues/1449)